### PR TITLE
Add space to end of tag value if it ends with a back slash

### PR DESCRIPTION
**Pull request by metal-hed**
Hoping to merge this change in to work around influxdb issue https://github.com/influxdata/influxdb/issues/5231

### CODE REVIEW

**Comment by repeatedly** [comment]
Do you report this problem on influxdb-ruby repository?

**Comment by metal-hed** [comment]
I did not, I can open an issue there if it is more appropriate

**Comment by repeatedly** [comment]
> I did not, I can open an issue

It should because we can merge your patch but it is workaround.
If influxdb has a problem, official client should handle it.

After you report the issue on influxdb-ruby repository, we will merge the patch.

File: lib/fluent/plugin/out_influxdb.rb [hunk 1]
```diff
@@ -138,7 +138,13 @@ def write(chunk)
           if (@auto_tags && v.is_a?(String)) || @tag_keys.include?(k)
             # If the tag value is not nil, empty, or a space, add the tag
             if v.to_s.strip != ''
-              tags[k] = v
+              # If tag value ends in a \, append a space at the end to avoid formatting error
+              if v.to_s.end_with?("\\")
+                log.info "Value '#{v}' ended in a \\, appending a space to the end of the string"
```
Comment (repeatedly) [inline lib/fluent/plugin/out_influxdb.rb]: `info` is good? It seems debug or trace level for me.

**Comment by metal-hed** [comment]
@repeatedly Bug reported to influxdb-ruby https://github.com/influxdata/influxdb-ruby/issues/225

I also updated the log message to `debug` level

**Comment by repeatedly** [comment]
Ah, I see.
We fixed influx-ruby version to support older environment before.

If latest version fixes many bugs, we can update dependency to latest version and release new fluent-plugin-influxdb as v2.0.

**Comment by dmke** [comment]
Just a heads up, the next `influxdb` gem release (containing this fix) will officially drop support for MRI 2.2 (reason being Bundler not supporting it anymore and no easy way do convince Travis to play nice here...).

I'm not actually removing 2.2 support, it's just that I can't test it anymore.

**Comment by metal-hed** [comment]
@repeatedly sounds like we have a work around to use. I will close this PR as it's not really needed anymore.

Thanks for the help!

**Comment by metal-hed** [comment]
> Ah, I see.
> We fixed influx-ruby version to support older environment before.
> 
> If latest version fixes many bugs, we can update dependency to latest version and release new fluent-plugin-influxdb as v2.0.

@repeatedly Are you able to do this in the near future? If not could you point me to some documentation to do this myself on a forked repo? Mainly just want to upgrade the influx-ruby version being used

**Pull request closed without merging**